storage/build.gradle.kts

```diff
@@ -0,0 +1,58 @@
+import java.io.ByteArrayOutputStream
+
+dependencies {
+    api(project(":infrastructure"))
+    with(rootProject.libs) {
+        implementation(akka.cluster.typed)
+        implementation(akka.cluster.sharding.typed)
+        implementation(akka.persistence.cassandra)
+        implementation(akka.persistence.typed)
+        implementation(akka.persistence.query)
+        testImplementation(akka.persistence.testkit)
+    }
+}
+
+tasks.withType<Test> {
+    doFirst {
+        exec {
+            workingDir(project.rootDir)
+            commandLine("docker", "compose", "up", "-d")
+        }
+
+        var healthy = false
+        var attempts = 0
+        val maxAttempts = 50
+
+        while (!healthy && attempts < maxAttempts) {
+            val output = ByteArrayOutputStream()
+            exec {
+                workingDir(project.rootDir)
+                commandLine("docker", "compose", "ps", "--format", "json")
+                isIgnoreExitValue = true
+                standardOutput = output
+            }
+
+            val outputStr = output.toString()
+            healthy = outputStr.contains("\"Health\":\"healthy\"")
+
+            if (!healthy) {
+                Thread.sleep(2000)
+                attempts++
+                println("Waiting for Cassandra to be healthy... Attempt ${attempts}/${maxAttempts}")
+            }
+
+
+        }
+
+        if (!healthy) {
+            throw GradleException("Cassandra failed to become healthy within timeout")
+        }
+    }
+
+    doLast{
+        exec {
+            workingDir(project.rootDir)
+            commandLine("docker", "compose", "down")
+        }
+    }
+}
```

Comment:
I found a nice [gradle plugin](https://github.com/avast/gradle-docker-compose-plugin) that embeds gradle compose commands in gradle tasks allowing us to configure tests more cleanly, waiting until all containers become healthy (and a lot of other options...)

Once you have configured it with
```
docker-compose = "com.avast.gradle.docker-compose:0.17.10"
```
you can simply:

```kotlin
dockerCompose.isRequiredBy(tasks.test)
```

Reply:
Really nice I'll try to use it